test/common/util.ts

```diff
@@ -7,6 +7,7 @@ import {TestContext} from "ava";
 export const baseUrl = "http://localhost:3000/helix-ui";
 
 export async function go(driver: WebDriver, component: string) {
+    // console.log(driver.get(`${baseUrl}/${component}`));
```

Comment:
Please remove this comment.

Reply:
Thank you Andrew. All comments are removed.